app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordsController < ApplicationController
+  
+  def new; end
+  
+  def create
+    user = User.where(email_address: params[:email_address]).first
```

Comment:
Prefer using `find_by` when you search for 1 record. The difference is `.first` method sorts table and performs 1 extra operation compared to `find_by` which uses `where().take` under the hood.